src/screens/LoginScreen/LoginScreen.styles.ts

```diff
@@ -1,34 +1,53 @@
 import { StyleSheet } from 'react-native';
 import * as colors from '@util/colors';
+import typography from '@util/typography';
+
+const SCREEN_HORIZONTAL_OFFSET = '5%';
 
 export default StyleSheet.create({
 	outerContainer: {
 		flex: 1,
+		flexBasis: '100%',
+	},
+	banner: {
```

Comment:
should 'banner' naming change to header? a banner usually means something else in UI

Reply:
Not sure... I only put on 'banner' because I couldn't think of a good name at the time.

Reply:
i mean it just semantics really, but banner's are usually the notification boxes that can be dismissed